lib/DDG/Spice/Hackage.pm

```diff
@@ -0,0 +1,16 @@
+package DDG::Spice::Hackage;
+# ABSTRACT: Search for information about Hackage packages
+
+use DDG::Spice;
+
+triggers startend => 'hask';
+
+spice to => 'http://www.haskell.org/hoogle/?mode=json&hoogle=$1';
```

Comment:
This redirects to `https://www.haskell.org/hoogle/?mode=json&hoogle=` (note the HTTPS part). Our Nginx config doesn't handle redirects at the moment, so we can't have any endpoints that redirect. Please change "HTTP" to "HTTPS".